python/lsst/afw/geom/ellipses/ellipse.cc

```diff
@@ -22,24 +22,36 @@
 
 #include <pybind11/pybind11.h>
 //#include <pybind11/operators.h>
-//#include <pybind11/stl.h>
+#include <pybind11/stl.h>
+
+#include "lsst/afw/geom/ellipses/BaseCore.h"
+#include "lsst/afw/geom/Point.h"
+#include "lsst/afw/geom/ellipses/Ellipse.h"
 
 namespace py = pybind11;
+using namespace pybind11::literals;
```

Comment:
This is for the `"name"_a` syntax used to define kwargs, so I think we need this using declaration.  Might be good to add a comment to that effect, though if we have it in every file that will quickly become common knowledge.